datafusion/src/logical_plan/expr.rs

```diff
@@ -1086,9 +1086,9 @@ unary_scalar_expr!(Trunc, trunc);
 unary_scalar_expr!(Abs, abs);
 unary_scalar_expr!(Signum, signum);
 unary_scalar_expr!(Exp, exp);
-unary_scalar_expr!(Log, ln);
```

Comment:
perhaps we should keep a `log` entry / alias to `log10` for postgres compatibility?

Reply:
i guess it's kept at the https://github.com/apache/arrow-datafusion/pull/271/files#diff-97178da6c90bfcb5e3dc80f06248ac1aebdec5a0e0e1d8ea235e2f571b5badf8R227 level, not here.

Reply:
here the macro defs shall not allow dups

Reply:
Definitely in favour of Postgres behaviour. 👍

Reply:
see https://github.com/apache/arrow-datafusion/pull/275